Assets/Scripts/Menu/DestroyBlocker.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class DestroyBlocker : MonoBehaviour {
+
+	// Update is called once per frame
+	void Update () {
+        Dropdown.Destroy(GameObject.Find("Blocker"));
```

Comment:
I delete this script because finally, just need to desactivate width force expand of the horizontal layout in the canvas.
So, as I modified MainScene.unity, I made a new commit 4d018a6.